Tasks/SonarQubePostTest/CodeAnalysisFilePathComputation.ps1

```diff
@@ -1,231 +0,0 @@
-. ./SonarQubePostTestImpl.ps1
```

Comment:
At this point this is dead code as it's handling the analysis report for PR-CA. I'll bring it back as part of the PR-CA work.